src/routes/(console)/(migration-wizard)/wizard.svelte

```diff
@@ -1,52 +1,409 @@
+    import ResourceForm from './resource-form.svelte';
     import { requestedMigration } from '$routes/store';
+    import { formData, provider, selectedProject, selectedRegion } from '.';
+    import { ID, type Models, Query } from '@appwrite.io/console';
+    import { InputSelect, InputText } from '$lib/elements/forms';
+    import {
+        Button,
+        Card,
+        Fieldset,
+        Icon,
+        Input,
+        Layout,
+        Skeleton,
+        Spinner,
+        Typography
+    } from '@appwrite.io/pink-svelte';
+    import { isCloud } from '$lib/system';
+    import { regions } from '$lib/stores/organization';
+    import { getFlagUrl } from '$lib/helpers/flag';
+    import { addNotification } from '$lib/stores/notifications';
+    import { goto, invalidate, invalidateAll } from '$app/navigation';
     import { base } from '$app/paths';
-    import { sdk } from '$lib/stores/sdk';
+    import { migrationFormToResources } from '$lib/stores/migration';
+    import { EyebrowHeading } from '$lib/components';
+    import {
+        IconAppwrite,
+        IconCog,
+        IconCurrencyDollar,
+        IconExclamation,
+        IconGithub,
+        IconTrendingUp
+    } from '@appwrite.io/pink-icons-svelte';
+    import { Dependencies } from '$lib/constants';
 
     const onExit = () => {
         formData.reset();
         requestedMigration.set(null);
     };
 
+    const organizations = page.data.organizations
+        .teams as Models.TeamList<Models.Preferences>['teams'];
+
+    let showExitModal = false;
+    let showResources = false;
+    let loadingProjects = false;
+    let creatingProject = false;
+    let errorInResources = false;
+    let migrationStarted = false;
+    let projectSdkInstance: ReturnType<typeof sdk.forProject> | null = null;
+
+    let projects = [] as Models.ProjectList['projects'];
+    let selectedOrg = organizations.length ? organizations[0].$id : null;
+
+    let newProjName = '';
+    let projectType: 'existing' | 'new' = 'existing';
+
+    async function getProjects(orgId: string | null) {
+        if (!orgId) {
+            projects = [];
+        } else {
+            loadingProjects = true;
+            projects = await sdk.forConsole.projects
+                .list([Query.equal('teamId', orgId), Query.orderDesc('$createdAt')])
+                .then((res) => res.projects);
+
+            loadingProjects = false;
+            projectType = projects.length ? 'existing' : 'new';
+            if (projectType === 'existing') {
+                const first = projects[0];
+                $selectedProject = first.$id;
+                projectSdkInstance = sdk.forProject(first.region, first.region);
+            }
+        }
+    }
+
+    function getProjectName(): string {
+        return isExisting ? getSelectedProject().name : newProjName || 'Appwrite project';
+    }
+
+    function getSelectedProject() {
+        return projects.find((project) => project.$id === $selectedProject);
+    }
```

Comment:
I think it might be better to use a dynamic `$: project = ...` value instead of calling the same function multiple times :thinking: